pcap-int.h

```diff
@@ -271,6 +271,19 @@ struct pcap {
 	get_airpcap_handle_op_t get_airpcap_handle_op;
 #endif
 	cleanup_op_t cleanup_op;
+
+#ifdef HAVE_REMOTE
```

Comment:
This should really be in a private data structure for remote captures, pointed to by the `priv` member of a `struct pcap`.